pkg/chartverifier/helmcertifier.go

```diff
@@ -31,9 +31,5 @@ type CertifierBuilder interface {
 }
 
 type Certifier interface {
-	Certify(uri string) (Certificate, error)
-}
-
-type Certificate interface {
-	IsOk() bool
+	Certify(uri string) (*Certificate, error)
```

Comment:
it looks strange to me that `Certificate` can certify something. I would expect that a certificate is result of some done certification. BTW, I would stop calling this certificate and start using Report or something else.

Reply:
Use of certificate instead of report goes way beyond this PR. Have opened a new issue for that work: #78